plugins/org.eclipse.elk.core/src/org/eclipse/elk/core/Core.melk

```diff
@@ -833,6 +833,87 @@ programmatic option scaleFactor: double {
 // requires hierarchyHandling != HierarchyHandling.INCLUDE_CHILDREN 
 }
 
+advanced option topdownLayout: boolean {
+    label "Topdown Layout"
+    description
+        "Turns topdown layout on and off. If this option is enabled, hierarchical layout will be computed first for
+         the root node and then for its children recursively. Layouts are then scaled down to fit the area provided by
+         their parents. Graphs must follow a certain structure for topdown layout to work properly. 
+         {@link TopdownNodeTypes.PARALLEL_NODE} nodes must have children of type 
+         {@link TopdownNodeTypes.HIERARCHICAL_NODE} and must define {@link topdown.hierarchicalNodeWidth} and 
+         {@link topdown.hierarchicalNodeAspectRatio} for their children. Furthermore they need to be laid out using the 
+         algorithm {@link TopdownPacking}. Hierarchical nodes require the layout option {@link nodeSize.fixedGraphSize}
+         to be true to prevent the algorithm used there from resizing the hierarchical node."
+    default = false
+    targets parents
+    requires topdown.nodeType
+}
+
+group topdown {
+    programmatic option scaleFactor: double {
+        label "Topdown Scale Factor"
+        description
+            "The scaling factor to be applied to the nodes laid out within the node in recursive topdown
+             layout. The difference to 'Scale Factor' is that the node itself is not scaled. This value has to be set on 
+             hierarchical nodes."
+        lowerBound = ExclusiveBounds.greaterThan(0)
+        default = 1
+        targets parents
+        requires nodeType == TopdownNodeTypes.HIERARCHICAL_NODE
+    }
+    
+    advanced option hierarchicalNodeWidth: double {
+        label "Topdown Hierarchical Node Width"
+        description
+            "The fixed size of a hierarchical node when using topdown layout. This value has to be set on parallel 
+             nodes."
+        default = 100
+        targets parents
+        requires nodeType == TopdownNodeTypes.PARALLEL_NODE
+    }
+    
+    advanced option hierarchicalNodeAspectRatio: double {
+        label "Topdown Hierarchical Node Aspect Ratio"
+        description
+            "The fixed aspect ratio of a hierarchical node when using topdown layout. Default is 1:sqrt(2). This value 
+             has to be set on parallel nodes."
+        default = 1.414
```

Comment:
If a default value is set here, it should only be set in other *.melk files if there is a good reason for a different default for that algorithm.